integration-cli/docker_cli_start_volume_driver_unix_test.go

```diff
@@ -129,11 +129,11 @@ func (s *DockerExternalVolumeSuite) SetUpSuite(c *check.C) {
 		fmt.Fprintln(w, `{}`)
 	})
 
-	if err := os.MkdirAll("/usr/share/docker/plugins", 0755); err != nil {
+	if err := os.MkdirAll("/etc/docker/plugins", 0755); err != nil {
```

Comment:
the `/usr/share` path is completely gone, we are not going to make this stable with that path.